src/app/modules/login/login.component.ts

```diff
@@ -1,12 +1,33 @@
 import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../../../backend-client';
 
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss'],
 })
 export class LoginComponent implements OnInit {
-  constructor() {}
+  email = '';
+  password = '';
+
+  constructor(private api: ApiService) {}
 
   ngOnInit(): void {}
+
+  onSubmit() {
+    return this.api
+      .authControllerLogin({
+        email: this.email,
+        password: this.password,
+      })
+      .subscribe(
+        (result) => {
+          // console.log(result)
+          localStorage.setItem('accessToken', result.accessToken);
```

Comment:
this should be AuthService. instead of `localStorage.setItem('accessToken', result.accessToken)`
It should be something like this
```
class AuthService {
   private token    
   currentUser: Subject
   setCurrentUser(user)   
   setToken(token);
   getToken();    
}
```

example here: https://jasonwatmore.com/post/2019/06/22/angular-8-jwt-authentication-example-tutorial#authentication-service-ts

feel free to copy but please adapt the code base on our use case